projects/dashboards/ecometrics/src/app/app.component.html

```diff
@@ -0,0 +1,5 @@
+<h1 class="mat-h1">
+    <img src="assets/geoengine.svg" class="logo" />
+    <div>ECOMETRICS</div>
+</h1>
+<app-dashboard></app-dashboard>
```

Comment:
![grafik](https://github.com/user-attachments/assets/2a0eea66-889c-4382-a376-2f6427ab3f31)

Reset does not reset

Reply:
8f0db5fceb446f85e1e9e034e83274c3698d8877